Language/pt-BR/CLI.php

```diff
@@ -38,19 +39,10 @@
 
 return [
 	'commandNotFound'     => 'Comando "{0}" não foi encontrado.',
-	'altCommandSingular'  => 'Você quis dizer isso?',
```

Comment:
You are removing indexes that are in the current CodeIgniter4 repository.
The pt-BR language is synchronized with the new version. Please, remove it.

Reply:
Or rather, do not remove it.

Reply:
I cant find at En lang 

https://github.com/codeigniter4/framework/blob/master/system/Language/en/CLI.php

Reply:
Found here: https://github.com/codeigniter4/CodeIgniter4/blob/develop/system/Language/en/CLI.php

Reply:
You are correct, but the reference for the current translate is the stable version.

Let's wait for comments / reviews from @lonnieezell

Reply:
If tested, the results are required to remove the extra key to pass the test. 
Like comment @lonnieezell at #127.